scripts/cibuild

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    GIT_COMMIT="$(git rev-parse --short HEAD)"
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Build application for staging or release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+    make clean
+    ./scripts/test
+    make assembly
```

Comment:
Do you think it makes sense to run `sbt build` inside of a container, and stop installing Java in the VM?

```diff
diff --git a/docker-compose.test.yml b/docker-compose.test.yml
index 80afb04..c094992 100644
--- a/docker-compose.test.yml
+++ b/docker-compose.test.yml
@@ -3,6 +3,9 @@ version: '2.1'
 services:
   gtsite-service:
     image: gtsite-service:${GIT_COMMIT}
+    build:
+      context: ./service
+      dockerfile: Dockerfile
 
   gtsite-nginx:
     image: gtsite-nginx:${GIT_COMMIT}
\ No newline at end of file
diff --git a/docker-compose.yml b/docker-compose.yml
index d73ea11..f7a022a 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -2,16 +2,17 @@ version: '2.1'
 
 services:
   gtsite-service:
-    image: quay.io/geotrellis/gtsite-service
-    build:
-      context: ./service
-      dockerfile: Dockerfile
+    image: openjdk:8-jre
     ports:
       - "9090:9090"
+    command: java -jar /srv/site.jar
+    working_dir: /opt/service
     volumes:
       - "${HOME}/.ivy2:/root/.ivy2"
       - "${HOME}/.m2:/root/.m2"
       - "${HOME}/.sbt:/root/.sbt"
+      - ./service:/opt/service
+      - ./service/srv/target/scala-2.10/site.jar:/srv/site.jar
diff --git a/scripts/cibuild b/scripts/cibuild
index db2f94d..9c98c2c 100755
--- a/scripts/cibuild
+++ b/scripts/cibuild
@@ -24,10 +24,11 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
 
-    make clean
-    make assembly
-    make assets
-    GIT_COMMIT="${GIT_COMMIT}" make build
+        docker-compose build gtsite-service
+        docker-compose run --rm --entrypoint ./sbt gtsite-service update
+        docker-compose run --rm --entrypoint ./sbt gtsite-service assembly
+        docker-compose -f docker-compose.yml \
+                       -f docker-compose.test.yml build gtsite-service
```